packages/gradient_boosting_model/gradient_boosting_model/processing/features.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from gradient_boosting_model.processing.errors import InvalidModelInputError
+
+
+class LogTransformer(BaseEstimator, TransformerMixin):
```

Comment:
We shouldn't have to apply a log transform to the variables if using gradient boosting machines. This was for the linear regression. Why is this here?

Reply:
removed